Article/Article.less

```diff
@@ -11,6 +11,7 @@
   height: 50px;
   overflow: hidden;
   box-shadow: 2px 2px 2px lightgrey;
+  transition: height .4s;
```

Comment:
Very easy animation, right? But it makes a big difference. Compare the difference between how the article opens vs the menu.